v2/bigquery-to-bigtable/src/test/java/com/google/cloud/teleport/v2/templates/BigQueryToBigtableIT.java

```diff
@@ -143,4 +129,66 @@ public void testBigQueryToBigtable() throws IOException {
                   });
         });
   }
+
+  @Test
+  public void testBigQueryToBigtableWithQuery() throws IOException {
+    String tableName = "test_table";
+    String colFamily = "names";
+
+    PipelineLauncher.LaunchConfig.Builder options =
+        PipelineLauncher.LaunchConfig.builder(testName, specPath)
+            .addParameter(
+                READ_QUERY,
+                "SELECT * FROM `"
+                    + String.join(".", PROJECT, bigQueryClient.getDatasetId(), tableName)
```

Comment:
nit: this `String.join` could be done in a utility method like we have `toTableSpec(table)`

Reply:
Done.